docs/cli.md

```diff
@@ -29,8 +34,281 @@ Common Flags
 | --insecure-skip-tls-verify | Skip SSL certificate validation (will make HTTPS insecure) |
 | --help (-h)                | Display help for specified command |
 
-oc get
------------
+## Basic Commands
+
+### oc
+
+This displays the list of available commands.
+
+### oc login
+
+This retrieves a session token that allows you to act as a user.
+Invoked without args, `oc login` prompts for a username and a password.
+For example, these two invocations are identical:
+
+```bash
+$ oc login
+Username: test
+Password: test
+
+$ oc login -u test -p test
+```
+
+See also [`oc logout`](#oc-logout) and [`oc whoami`](#oc-whoami).
+
+### oc new-project
+
+This creates a new project, with you as the project admin.
+Option `--display-name` specifies the user-facing name of the project.
+Option `--description` specifies its description.
+
+For example:
+
+```
+$ oc new-project web-team-dev \
+  --display-name="Web Team Development" \
+  --description="Development project for the web team."
+```
+
+Note that we use double-quotes around the option arguments.
+
+### oc new-app
+
+This is a superset of [`oc new-build`](#oc-new-build).
```

Comment:
I am not sure we should present `new-app` as a superset of `new-build`. Have a look at the command [description](https://github.com/openshift/origin/blob/f21a80e484e2ccc705a74a815df4a48e536c1acc/pkg/cmd/cli/cmd/newapp.go#L32-L45) and reword this likewise or use that as is.